frontend/app/model/Eventbrite.scala

```diff
@@ -207,10 +207,17 @@ object Eventbrite {
     val limitedAvailabilityText = "Last tickets remaining"
     val isLimitedAvailability = internalTicketing.exists(event => event.ticketsNotSold <= Config.eventbriteLimitedAvailabilityCutoff && !event.isSoldOut)
     val ticketsNotSold = internalTicketing.map(_.ticketsNotSold)
-
     val isSoldOut = internalTicketing.exists(_.isSoldOut)
-    val isBookable = status == "live" && !isSoldOut
-    val isPastEvent = status != "live" && status != "draft"
+
+    val isBookable = {
+      val hasBookableTicketClasses = ticket_classes.exists(_.sales_end < DateTime.now)
+      (status == "live" || (status == "started" && hasBookableTicketClasses)) && !isSoldOut
+    }
+
+    val isPastEvent = {
+      val conditions = Set("cancelled", "ended", "completed")
```

Comment:
Random comment: I'm not sure if a cancelled event _is_ a past event... (obviously, it probably gets cancelled before it was supposed to happen). It's like what we mean is 'an event which we are no longer going to attempt to sell tickets for'.